tests/test_client.py

```diff
@@ -0,0 +1,69 @@
+import pytest, json
+from unittest.mock import Mock
+from app import create_app
+from managers.slack_manager import SlackManagerError
+
+
+@pytest.fixture()
+def app():
+    app = create_app()
+    yield app
+
+
+def test_get_channels_status(app):
+    slack_manager = Mock()
+    app.config["SlackManager"] = slack_manager
+    sample_channels = [
+        ["CB8886H7V", "random"],
+        ["CB98179M4", "general"],
+        ["CEQ4SQQAF", "blog_posts"],
+    ]
+    slack_manager.get_channel_list.return_value = sample_channels
+    response = app.test_client().get("/channels")
+    assert json.loads(response.data.decode()) == sample_channels
+
+    response = app.test_client().get("/status")
+    assert json.loads(response.data.decode()) == {
+        "status": "Slack API is connected and working properly"
+    }
+
+    slack_manager.get_channel_list.side_effect = SlackManagerError("test_error")
+    response = app.test_client().get("/channels")
+    assert json.loads(response.data.decode()) == {
+        "error": "Slack exception ocurred: test_error"
+    }
+
+    response = app.test_client().get("/status")
+    assert json.loads(response.data.decode()) == {
+        "error": "Slack exception ocurred: test_error"
+    }
+
+
+def test_post_message(app):
```

Comment:
for these tests where we check multiple inputs / expected outputs, can we use a parameterize fixture? https://docs.pytest.org/en/6.2.x/parametrize.html